tests/ios/app/BUILD.bazel

```diff
@@ -280,3 +281,5 @@ ios_application(
         ":SwiftLib",
     ],
 )
+
+make_tests()
```

Comment:
is it that we can add more tests under this method in future?

Reply:
Mainly a target can't be instantiated from a `.bzl` file, and I went off the convention of putting the analysis test definitions adjacent to the impl in the `.bzl` file to keep the BUILD file minimal. Also a nice place to put more instances of `_identical_outputs_test` e.g. for app VS test transitive dep consistency.